databricks/koalas/plot.py

```diff
@@ -382,7 +382,7 @@ def rc_defaults(
         showcaps=None,
         showbox=None,
         showfliers=None,
-        **kwargs
+        **kwargs,
```

Comment:
Hmm, why did we come to need the `,` at the end?

Reply:
When I run  `PYTHON_EXECUTABLE=python ./dev/reformat `, that automatically gets added.